src/platform/site-wide/sass/modules/_m-facilities.scss

```diff
@@ -0,0 +1,57 @@
+// Facilities Hub
+.facility-main-cta {
+  margin: 8px 0;
+}
+
+.facility-main-cta-item {
+  border-left: 4px solid $color-link-default;
+  background-color: $color-primary-alt-lightest;
+  height: 40px;
+  margin-right: 14px;
+  padding-top: 5px;
+  padding-left: 14px;
+  &:last-child {
+    margin-right: 0;
+  }
+  a {
+    color: #3f77b0;
+    text-decoration: none;
+  }
+}
+.facility-hub-cta-item {
+  border-top: 1px solid $color-primary-alt-light;
+  height: 73px;
+  line-height: 73px;
+  position: relative;
+
+  span {
```

Comment:
It'd be better to either create a class for these elements (the span here and the a below) or use `> span` to limit how broadly these styles apply, I think.